src/Platform/Microsoft.Testing.Platform/Configurations/AggregatedConfiguration.cs

```diff
@@ -58,13 +58,6 @@ public string? this[string key]
     public void SetTestHostWorkingDirectory(string workingDirectory) =>
         _testHostWorkingDirectory = Guard.NotNull(workingDirectory);
 
-    public void CreateDefaultTestResultDirectory()
```

Comment:
@MarcoRossignoli do you recall if that's just dead code or something that got broken?

Reply:
The method was introduced in #2389 and it was unused in that PR